Scripts/DCS-BIOS/lib/modules/aircraft_modules/F4U-1D.lua

```diff
@@ -209,7 +209,10 @@ F4U_1D:defineFloat("AIRSPEED_INDICATOR", 11, { 0, 1 }, ASI, "Airspeed Indicator"
 -- local HORIZON_GYRO = "Gyro Horizon"
 
 -- Climb Indicator
--- local VSI = "Climb Indicator"
+local VSI = "Climb Indicator"
+
+F4U_1D:defineFloat("CLIMB_INDICATOR", 29, { -1, 1 }, VSI, "Climb Indicator")
+F4U_1D:definePotentiometer("CLIMB_INDICATOR_SET", devices.NAV, 3302, 229, { -1, 1 }, VSI, "Climb Indicator Set")
```

Comment:
Looks like this is commented out in the luas - since there's no clickable for it currently, I wouldn't map it. We can always add it later.

It's also hard to know if this will be a potentiometer or more likely an encoder anyway.

```suggestion
```